<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qwCanvas</class>
 <widget class="QWidget" name="qwCanvas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qwCanvas</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pbExecution">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>720</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Execution</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbInitial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>720</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Initial</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>660</y>
     <width>337</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QSpinBox" name="sbStartX">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="sbStartY">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="sbEndX">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="sbEndY">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="dsbBiasX">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QDoubleSpinBox" name="dsbBiasY">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="dsbCorrectedX">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QDoubleSpinBox" name="dsbCorrectedY">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="dsbRobotX">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QDoubleSpinBox" name="dsbRobotY">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string> X: </string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Current: </string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Corrected: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Bias: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Destination:   </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Start position:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Thetaï¼š</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="sbStartTheta">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QDoubleSpinBox" name="dsbBiasTheta">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QDoubleSpinBox" name="dsbCorrectedTheta">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Y: </string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QDoubleSpinBox" name="dsbRobotTheta">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pbExecution</sender>
   <signal>clicked(bool)</signal>
   <receiver>qwCanvas</receiver>
   <slot>execution(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbInitial</sender>
   <signal>clicked()</signal>
   <receiver>qwCanvas</receiver>
   <slot>initialDirection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>initialDirection()</slot>
  <slot>execution(bool)</slot>
 </slots>
</ui>
